.github/workflows/cd.yaml

```diff
@@ -0,0 +1,69 @@
+name: Deploy to ECS
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  AWS_REGION: eu-central-1
+  ECR_REPOSITORY_NAME: cloud-auth
+  ECS_CLUSTER_NAME: cloud-auth
+  ECS_SERVICE_NAME: cloud-auth-service
+  TASK_FAMILY_NAME: cloud-auth
+  PORT: 3333
+  CLOUD_APP_ORIGIN: ${{ secrets.CLOUD_APP_ORIGIN }}
+  SUPABASE_JWT_SECRET: ${{ secrets.SUPABASE_JWT_SECRET }}
+  COOKIE_NAME: ${{ secrets.COOKIE_NAME }}
+  COOKIE_SECRET: ${{ secrets.COOKIE_SECRET }}
+  DIRECT_URL: ${{ secrets.DIRECT_URL }}
+  DATABASE_URL: ${{ secrets.DATABASE_URL }}
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 1.1.0
+
+      - name: Initialize Terraform
+        run: terraform init
+
+      - name: Log in to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+        with:
+          region: ${{ env.AWS_REGION }}
+
+      - name: Build and push Docker image to Amazon ECR
```

Comment:
You can use docker build here, have a look at Echo's deploy here: https://github.com/WalletConnect/echo-server/blob/85b8c485a4645ee231de8662b6875809ab775837/.github/workflows/release.yml#L110-L137